py/torch_tensorrt/dynamo/conversion/impl/deconv.py

```diff
@@ -140,12 +145,7 @@ def deconvNd(
         # stride in deconv1d is (2,) -> need to change to (2, 1) in deconv2d
         stride = (stride[0], 1) if stride is not None else stride
         dilation = (dilation[0], 1) if dilation is not None else dilation
-        output_padding = (
-            (tuple(output_padding) + (0,))
-            if output_padding is not None
-            else output_padding
-        )
-
+        output_padding = tuple(output_padding) + (0,)
```

Comment:
Doesnt this change the code to now instead of it being `output_padding + (0,) | None` it will be `output_padding + (0,) | (None, 0)` ? Does not seem to be the same